challenge.rb

```diff
@@ -57,7 +67,26 @@ def ac_monitor current_temp, ac_working, desired_temp
 # - return "fizzbuzz" if `num` is evenly divisible by both 3 and 5
 # - return the value of `num` if `num` isn't divisible by either 3 or 5
 
-# Replace this comment with your code.
+ def fizz_buzz_calculator(num
+    if num%5 == 0 && num%3 == 0
+      puts "Fizzbuzz"
+
+    elsif num%3 == 0
+      puts 'Fizz'
+
+    elsif num%5 == 0
+      puts 'Buzz'
+
+    else
+      puts i
+
+    end
+    i += 1
+  end
+
+  return list
```

Comment:
This method should be returning one of the four values; there's no need to iterate or generate a list. Your fizzbuzz logic is right, though.